Bazel CI: Kythe is failing in downstream with Bazel@HEAD - Error Prone

https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/2856#01862f36-b9c9-416b-81dc-a9a5c0ff2dc3

```
(04:14:07) ERROR: /var/lib/buildkite-agent/builds/bk-docker-222l/bazel-downstream-projects/kythe/kythe/java/com/google/devtools/kythe/extractors/shared/BUILD:35:13: Building kythe/java/com/google/devtools/kythe/extractors/shared/libfile_vnames.jar (1 source file) failed: (Exit 1): java failed: error executing command (from target //kythe/java/com/google/devtools/kythe/extractors/shared:file_vnames)
--
  | (cd /var/lib/buildkite-agent/.cache/bazel/_bazel_buildkite-agent/0d48e037abb2184fadf94785455d1155/execroot/io_kythe && \
  | exec env - \
  | LC_CTYPE=en_US.UTF-8 \
  | PATH=/bin:/usr/bin:/usr/local/bin \
  | external/remotejdk17_linux/bin/java '--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED' '--add-exports=jdk.compiler/com.sun.tools.javac.main=ALL-UNNAMED' '--add-exports=jdk.compiler/com.sun.tools.javac.model=ALL-UNNAMED' '--add-exports=jdk.compiler/com.sun.tools.javac.processing=ALL-UNNAMED' '--add-exports=jdk.compiler/com.sun.tools.javac.resources=ALL-UNNAMED' '--add-exports=jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED' '--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED' '--add-opens=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED' '--add-opens=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED' '--add-opens=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED' '--add-opens=jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED' '--add-opens=java.base/java.nio=ALL-UNNAMED' '--add-opens=java.base/java.lang=ALL-UNNAMED' '-Dsun.io.useCanonCaches=false' -XX:-CompactStrings -jar external/remote_java_tools/java_tools/JavaBuilder_deploy.jar @bazel-out/k8-fastbuild/bin/kythe/java/com/google/devtools/kythe/extractors/shared/libfile_vnames.jar-0.params @bazel-out/k8-fastbuild/bin/kythe/java/com/google/devtools/kythe/extractors/shared/libfile_vnames.jar-1.params)
  | # Configuration: dfc7da9c2e0f38ef6c6132bdb907cd347f7d84a79d1b8116658f09144e419883
  | # Execution platform: @local_config_platform//:host
  | kythe/java/com/google/devtools/kythe/extractors/shared/FileVNames.java:235: error: [UnusedMethod] Method 'fillInWith' is never used.
  | public VName fillInWith(Matcher m, String defaultCorpus) {
  | ^
  | (see https://errorprone.info/bugpattern/UnusedMethod)
  | Did you mean to remove this line?
  | (04:14:07) INFO: Elapsed time: 75.821s, Critical Path: 13.01s
  | (04:14:07) INFO: 14491 processes: 7095 remote cache hit, 7258 internal, 20 linux-sandbox, 111 local, 7 worker.
  | (04:14:07) ERROR: Build did NOT complete successfully
  | bazel build failed with exit code 1
  | 🚨 Error: The command exited with status 1
```

--- a/kythe/java/com/google/devtools/kythe/extractors/shared/FileVNames.java
+++ b/kythe/java/com/google/devtools/kythe/extractors/shared/FileVNames.java
@@ -228,14 +228,6 @@ public class FileVNames {
       return b.build();
     }
 
-    /**
-     * Returns a {@link VName} by filling in its corpus/root/path with regex groups in the given
-     * {@link Matcher}.
-     */
-    public VName fillInWith(Matcher m, String defaultCorpus) {
-      return fillInWith(m, () -> defaultCorpus);
-    }
-
     private static final Pattern replacerPattern = Pattern.compile("@(\\w+)@");
 
     private static String fillIn(String tmpl, Matcher m) {

